app/services/testdata/StatusGeneratorFactory.scala

```diff
@@ -58,6 +58,14 @@ object StatusGeneratorFactory {
         }
       case (PHASE1_TESTS, Some(PHASE1_TESTS_COMPLETED)) => Phase1TestsCompletedStatusGenerator
       case (PHASE1_TESTS, Some(PHASE1_TESTS_RESULTS_RECEIVED)) => Phase1TestsResultsReceivedStatusGenerator
+      case (PHASE2_TESTS, Some(PHASE2_TESTS_INVITED)) => Phase2TestsInvitedStatusGenerator
+      case (PHASE2_TESTS, Some(PHASE2_TESTS_STARTED)) => Phase2TestsStartedStatusGenerator
+      case (PHASE2_TESTS, Some(PHASE2_TESTS_EXPIRED)) =>
+        if (generatorConfig.phase1StartTime.isDefined) {
```

Comment:
Dirty hack to avoid going over the 22 attribute limit for the unapply method.  Rationale is that if you're generating a phase2 test then does it really matter when the phase1 was started as it should already be complete and passed.  We should probably redo this to pass in a JSON object or something.